charts/console/values.yaml

```diff
@@ -213,30 +210,25 @@ secret:
     # tlsCert:
     # tlsKey:
     # tlsPassphrase:
-    # schemaRegistryPassword:
-    # schemaRegistryTlsCa:
-    # schemaRegistryTlsCert:
-    # schemaRegistryTlsKey:
-    # protobufGitBasicAuthPassword
+   
+  schemaRegistry: {}
+    # password:
+    # tlsCa:
+    # tlsCert:
+    # tlsKey:
+
   # Enterprise version secrets
   # - SSO secrets (Enterprise version).
-  login:
+  authentication:
     # Configurable JWT value
-    jwtSecret: ""
-    google: {}
-      # clientSecret:
-      # groupsServiceAccount:
-    github: {}
-      # clientSecret:
-      # personalAccessToken:
-    okta: {}
-      # clientSecret:
-      # directoryApiToken:
+    jwtSigningKey: ""
     oidc: {}
       # clientSecret:
 
-  enterprise: {}
-    # license:
+  serde: {}
+    # protobufGitBasicAuthPassword
+
+  license: ""
```

Comment:
The change log mentioned that this was now a top level key but it's nested under `secret` is that intended?

Reply:
The changelog was misleading. The values is correct. It was moved from `secret.enterprise.license` to `secret.license`.